website/source/docs/platform/k8s/index.html.md

```diff
@@ -63,17 +65,7 @@ There are several ways to try Consul with Kubernetes in different environments.
  - The [Consul and Kubernetes Deployment](
    https://learn.hashicorp.com/consul/day-1-operations/kubernetes-deployment-guide?utm_source=consul.io&utm_medium=docs) guide covers the necessary steps to install and configure a new Consul cluster on Kubernetes in production.
 
-## "consul-k8s" Project
```

Comment:
Since this documentation doesn't seem important to users I thought it best to remove it. Especially from the index page.

Reply:
How do we know it's not important to users?

Reply:
My thinking is that most users will install the helm chart and use the values.yaml to configure their installation. So when they first come to this page, they don't need to know about the consul-k8s repo.